docs/kb/semgrep-appsec-platform/missing-pr-comments.md

```diff
@@ -16,11 +16,10 @@ PR or MR comments are currently supported for:
 
 * All GitHub plans
 * All GitLab plans
-* Bitbucket Cloud
+* Bitbucket Cloud and Bitbucket Data Center
```

Comment:
Are there any Bitbucket plans remaining that we don't support?

Reply:
This is true; we support all Bitbucket options now!